apps/web/src/containers/App/App.tsx

```diff
@@ -269,6 +269,7 @@ export const App = () => {
   const {
     state: { themeVariant },
   } = globalState;
+
   const notionalTheme = useNotionalTheme(themeVariant);
 
   return (
```

Comment:
You don't need the Intercom Provider in here anymore?

Reply:
Good call, it must have gotten washed out when I rebased